ext/opentelemetry-ext-wsgi/src/opentelemetry/ext/wsgi/__init__.py

```diff
@@ -33,8 +33,10 @@ class OpenTelemetryMiddleware:
 
     Args:
         wsgi: The WSGI application callable.
+        propagators: TODO
     """
 
+    # pylint: disable=missing-type-doc
```

Comment:
This is what we get for using the same lint check in the API and SDK/etc.

Reply:
What type doc is it complaining about? Under-documented parameters. Since the Middleware is public API, we should probably really add that documentation.